backend/app/notify/email.go

```diff
@@ -265,7 +189,7 @@ func (e *Email) Send(ctx context.Context, req Request) (err error) {
 			return nil
 		}
 		log.Printf("[DEBUG] send notification via %s, comment id %s", e, req.Comment.ID)
-		msg, err = e.buildMessageFromRequest(req, req.ForAdmin)
+		msg, err = e.buildReplyMessage(req, req.ForAdmin)
```

Comment:
generally speaking, refactoring of other parts, nor related to PR shouldn't be mixed in